scm/etc/Cheyenne_setup_intel.sh

```diff
@@ -4,9 +4,10 @@ echo "Setting environment variables for SCM-CCPP on Cheyenne with icc/ifort"
 
 #load the modules in order to compile the GMTB SCM
 echo "Loading intel and netcdf modules..."
+deactivate
```

Comment:
What happens if someone doesn't have the pylab environment loaded? Will this just do nothing?